frontend/src/components/Buttons/QueueButton.vue

```diff
@@ -1,214 +1,164 @@
+        </v-icon>
+      </tooltip-button>
     </template>
     <v-card>
       <v-list>
-        <v-list-item>
-          <v-avatar>
-            <blurhash-image
-              v-if="playbackManager.initiator"
-              :item="playbackManager.initiator" />
-            <v-icon v-else :icon="modeIcon" />
-          </v-avatar>
+        <v-list-item :title="sourceText">
+          <template #prepend>
+            <v-avatar>
+              <blurhash-image
+                v-if="playbackManager.initiator"
+                :item="playbackManager.initiator" />
+              <v-icon v-else :icon="modeIcon" />
+            </v-avatar>
+          </template>
 
-          <v-list-item-title>{{ sourceText }}</v-list-item-title>
-          <v-list-item-subtitle>
-            {{ getTotalEndsAtTime(playbackManager.queue) }} -
+          <template #subtitle>
+            {{ getTotalEndsAtTime(playbackManager.queue).value }} -
             {{
               $t('playback.queueItems', {
                 items: playbackManager.queue.length
               })
             }}
-          </v-list-item-subtitle>
+          </template>
 
-          <v-list-item-action>
+          <template #append>
             <like-button
               v-if="playbackManager.initiator && playbackManager.currentItem"
               :item="playbackManager.currentItem" />
-          </v-list-item-action>
-          <v-list-item-action class="mr-1">
             <item-menu
               v-if="playbackManager.initiator && playbackManager.currentItem"
               :item="playbackManager.currentItem" />
-          </v-list-item-action>
+          </template>
         </v-list-item>
       </v-list>
       <v-divider />
       <v-list class="overflow">
         <!-- We set an special property to destroy the element so it doesn't take resources while it's not being used.
         This is especially useful for really huge queues. -->
-        <draggable-queue v-if="menu || !destroy" class="ml-4" />
+        <draggable-queue v-if="menuModel || !destroy" class="ml-4" />
       </v-list>
       <v-spacer />
-      <v-card-actions class="d-flex justify-space-between">
-        <v-tooltip location="top">
-          <template #activator="{ on: tooltip }">
-            <v-btn icon v-on="tooltip" @click="playbackManager.stop">
+      <v-card-actions>
+        <v-tooltip :text="$t('playback.clearQueue')" location="top">
+          <template #activator="{ props: tooltip }">
+            <v-btn icon v-bind="tooltip" @click="playbackManager.stop">
               <v-icon>
                 <i-mdi-playlist-remove />
               </v-icon>
             </v-btn>
           </template>
-          <span>{{ $t('playback.clearQueue') }}</span>
         </v-tooltip>
-        <v-tooltip location="top">
-          <template #activator="{ on: tooltip }">
-            <v-btn icon disabled v-on="tooltip">
+        <v-tooltip :text="$t('playback.saveAsPlaylist')" location="top">
```

Comment:
Why not migrate this to the new component?

Reply:
Done